src/qmflows/packages/packages.py

```diff
@@ -240,7 +239,7 @@ def _unpack_results(self) -> None:
         # Failed to find or unpickle the .dill file; issue a warning
         if file_exc is not None:
             self._results = None
-            warn(f"{file_exc}, setting value to 'None'")
+            warn(f"{file_exc}, setting value to 'None'", category=QMFlows_Warning)
```

Comment:
I like that having a `QMFlows_Warning` allows handling the warning on the client code (e.g. qmflows-namd) if we want to